openupgradelib/openupgrade.py

```diff
@@ -329,21 +337,21 @@ def load_data(cr, module_name, filename, idref=None, mode="init"):
         if ext == ".csv":
             noupdate = True
             tools.convert_csv_import(
-                cr, module_name, pathname, fp.read(), idref, mode, noupdate
+                env_or_cr, module_name, pathname, fp.read(), idref, mode, noupdate
             )
         elif ext == ".yml":
             yaml_import(cr, module_name, fp, None, idref=idref, mode=mode)
```

Comment:
This one hasn't been changed, isn't it?

Reply:
Has been changed at [5bf1207c8cb7575fba4501aff4932c2cc952bf9f](https://github.com/odoo/odoo/commit/5bf1207c8cb7575fba4501aff4932c2cc952bf9f)

Reply:
`yaml_import` was removed from Odoo after 11 anyway.

Reply:
I know, but right now, the code will fail in 11-, as `cr` has no value for that case. Assign previously `cr = env_or_cr`, or directly put here `env_or_cr`.

Reply:
you are right, I added more at line 310